mediafile.py

```diff
@@ -840,30 +840,33 @@ class MP3DescStorageStyle(MP3StorageStyle):
     """Store data in a TXXX (or similar) ID3 frame. The frame is
     selected based its ``desc`` field.
```

Comment:
Expanding the docstring to cover the new parameters might be nice here.

Reply:
It's just an example, this class was already used for other frames in code and now it's more generic so it's possible to use it with most of frames that would need it.
I'm opposed to this as imo it will only make docs less clear.

Reply:
Oh god. I had to misread this.
Of course, I'll add docs for parameters. ;)
I don't know why but I had frame types in my mind.

Reply:
```python
class MP3DescStorageStyle(MP3StorageStyle):
    """Store data in a TXXX (or similar) ID3 frame. The frame is
    selected based its ``desc`` field.
    ``attr`` allows to specify name of data accessor property in the frame.
    Most of frames uses `text`.
    ``multispec`` specifies if frame data is ``mutagen.id3.MultiSpec`` 
    which means that the data is being packed in the list.
    """
```
What do you think about something like that?

Reply:
Sounds great!! :sparkles:

Reply:
Done ;D